app/controllers/receive_package.js

```diff
@@ -28,180 +30,187 @@ export default Ember.Controller.extend({
     ];
   }),
 
-  selectedGrade: Ember.computed("model", function(){
+  selectedGrade: Ember.computed("model", function() {
     var grade = this.get("model.grade");
-    return this.get("grades").filterBy('id', grade).get("firstObject");
+    return this.get("grades")
+      .filterBy("id", grade)
+      .get("firstObject");
   }),
 
-  offer: Ember.computed("model", function(){
+  offer: Ember.computed("model", function() {
     return this.get("store").peekRecord("offer", this.get("package.offerId"));
   }),
 
-  identifyDevice: Ember.on('init', function() {
+  identifyDevice: Ember.on("init", function() {
     var isAndroidDevice = this.get("cordova").isAndroid();
     this.set("isAndroidDevice", isAndroidDevice);
   }),
 
-  location: Ember.computed("locationId", function(){
+  location: Ember.computed("locationId", function() {
     return this.store.peekRecord("location", this.get("locationId"));
   }),
 
   locationId: Ember.computed("package", {
     get: function() {
-      return this.get("package.location.id") || this.get("package.packageType.location.id");
+      return (
+        this.get("package.location.id") ||
+        this.get("package.packageType.location.id")
+      );
     },
     set: function(key, value) {
```

Comment:
this setter can be removed, it's not doing anything apart from returning the value